jwst/assign_wcs/tests/test_pointing.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import numpy as np
+
+from jwst.assign_wcs import pointing
+from jwst.datamodels.image import ImageModel
+from astropy.modeling.models import Identity
+
+from .test_nircam import create_hdul
+
+
+def create_imaging_datamodel(v2_ref, v3_ref, va_scale):
+    hdul = create_hdul()
+    datamodel = ImageModel(hdul)
+    datamodel.meta.velocity_aberration.scale_factor = va_scale
+    datamodel.meta.wcsinfo.v2_ref = v2_ref
+    datamodel.meta.wcsinfo.v3_ref = v3_ref
+    return datamodel
```

Comment:
This could be a Pytest fixture, and it would be much clearer that the tests depend on it.